client/scss/utilities/_mixins.scss

```diff
@@ -230,13 +230,13 @@
     transform: translateZ(0); // Fixes scale/border-radius overflow bug: http://stackoverflow.com/a/31686781
     border-radius: $border-radius-unit;
     overflow: hidden;
-  }
 
-  #{$zoomElement} {
-    display: block;
-    width: 100%;
-    backface-visibility: hidden; // Fixes chrome bug where image shifts slightly and appears blurry during transition
-    transition: transform 200ms ease, opacity 500ms ease;
+    #{$zoomElement} {
```

Comment:
yep, that's intended - otherwise backface-visibility was being applied to all .images regardless of whether they needed it, which was creating unnecessary composite layers